R/util-MakeSnapshotHelpers.R

```diff
@@ -310,26 +310,26 @@ MakeRptKriDetails <- function(lResults, status_site, meta_workflow, status_workf
       pt_data_dt = NA_character_
     ) %>%
     select("study_id",
```

Comment:
Per comment above, all this manual renaming isn't ideal. Options: 
- capture all of this as a `mapping` object
- refactor the assessment functions to use the new naming
- update the data model to use the old naming 

cc: @samussiah